crates/plugins/src/lib.rs

```diff
@@ -9,13 +9,19 @@ pub use store::PluginStore;
 
 /// List of Spin internal subcommands
 pub(crate) const SPIN_INTERNAL_COMMANDS: &[&str] = &[
+    "template",
     "templates",
     "up",
     "new",
     "add",
+    "login",
     "deploy",
     "build",
     "plugin",
+    "plugins",
     "trigger",
     "external",
+    "doctor",
+    "registry",
```

Comment:
`oci` is still accepted too. Yes, I had forgotten about it too.

Reply:
added